Subtester/Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -64,4 +64,16 @@ android {
         ignoreAssetsPattern = "!.svn:!.git:!.ds_store:!*.scc:.*:!CVS:!thumbs.db:!picasa.ini:!*~"
     }**PACKAGING_OPTIONS**
 }**REPOSITORIES****SOURCE_BUILD_SETUP**
+
+// When using Unity Package Manager, this link will be wrong but still needed.
 **EXTERNAL_SOURCES**
+
+// IMPORTANT: If you're not installing through Unity Package Manager, remove the code below.
+// Symlink local sources. When using Unity's "Symlink local sources" checkbox,
+// The link will be wrong when using Unity Package Manager because of a bug in Unity.
+// That's not a problem if it points to a file that doesn't exist, but we need to add the path
+// that does exist here.
+// More info: https://forum.unity.com/threads/native-sources-inside-package-not-found-after-exporting-project-to-android.1355321/
+android.sourceSets.main.java.srcDirs += [
+    '**DIR_UNITYPROJECT**/../RevenueCat/Plugins/Android/PurchasesWrapper.java'
```

Comment:
Nice hack I like it

Reply:
Ahh nice! I guess we can leave the wrong path yeah 😅.

Also, we could make it `'**DIR_UNITYPROJECT**/../RevenueCat/Plugins/Android/*.java'` to include other files if we ever add any, but considering this is a hack until Unity fixes it (if ever) then I think this is ok.

Reply:
that's a good idea, I'll do that

Reply:
doing `*.java` wouldn't work, but I just made it `.....RevenueCat/Plugins/Android/` instead and it worked fine, going with that